network/application/MQTT_eclipse_paho/mqtt_noos_support.c

```diff
@@ -0,0 +1,187 @@
+/***************************************************************************//**
+ *   @file   mqtt_port_noos.c
```

Comment:
the file name here does not match the actual name of the file

Reply:
I don't understand the need to have this mqtt_noos_support and then we als have the mqtt_client.c one. Let's just add one file and include this functionality in that one. 

Also looking at the proposal for this:
noos/network/
├───application
│   │   mqqt_client.c
│   │   mqtt_client.h
│   │
│   └───MQTT_paho
│           mqtt_paho1.c
│           mqtt_paho1.h
│           mqtt_paho2.c
│           mqtt_paho2.h
│
├───data_link
│       at_params.h
│       at_parser.c
│       at_parser.h
│       wifi.c
│       wifi.h
│
├───network
│       network_private.h
│
└───transport
        tcp_socket.c
        tcp_socket.h

Reply:
In mqtt_client.h there is only what user need to use mqtt and in mqtt_noos_support is what paho MQTTClient need in order to work on noos.
so the dependencies will look like
app.h -> mqtt_client.h -> MQTTClient.h -> mqtt_noos_support.h